fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -221,12 +221,7 @@ public Cost visitPhysicalHashJoin(
         pattern2: (L join1 Agg1) join2 agg2
         in pattern2, join1 and join2 takes more time, but Agg1 and agg2 can be processed in parallel.
         */
-        double penalty = HEAVY_OPERATOR_PUNISH_FACTOR
-                * Math.min(probeStats.getPenalty(), buildStats.getPenalty());
-        if (buildStats.getWidth() >= 2) {
-            //penalty for right deep tree
-            penalty += rightRowCount;
-        }
+        double penalty = HEAVY_OPERATOR_PUNISH_FACTOR;
```

Comment:
I think CostModelV1 would be abandoned soon, and V2 doesn't need those fields

Reply:
1. this file is CostModelV1. if v2 will replace v1, keep the v1 untouched. it will work for next few days
2. in V2, we still need penalty, especially heavy operator penalty. only when the parallel parameter between child and parent is 0, the penalty is not used.

Reply:
V2 不需要统计信息提供这些字段

Reply:
> 1. this file is CostModelV1. if v2 will replace v1, keep the v1 untouched. it will work for next few days
> 2. in V2, we still need penalty, especially heavy operator penalty. only when the parallel parameter between child and parent is 0, the penalty is not used.

关于第一点我觉得本身代价模型就尚不成熟，另外如果后续发现影响较大可以再把这些逻辑补回来，目前V2的测试比较急，我觉得应该优先考虑V2的工作。流水线现在本身也不稳定且很慢，每次更新一次提交都很耽误时间